Web/Cube/src/WebARCube.tsx

```diff
@@ -1,83 +1,126 @@
+        }
+    }, [rotation]);
+
+    useEffect(() => {
+        if (scale) {
+            setAttributes(scale, props.mxScaleXAtt, props.mxScaleYAtt, props.mxScaleZAtt);
+        }
+    }, [scale]);
+
+    return (
+        <>
+            <GizmoComponent
+                color={props.mxGizmoColor?.value ?? "#ffffff"}
+                mesh={rotationSet ? mesh : undefined}
+                gizmoSize={Number(props.mxGizmoSize.value) ?? 0.05}
+                draggingEnabled={props.mxDraggingEnabled.value ?? false}
+                pinchEnabled={props.mxPinchEnabled.value ?? false}
+                rotationEnabled={props.mxPinchRotationEnabled.value ?? false}
+                onScale={newScale => {
+                    setScale(newScale);
+                }}
+                onDrag={newPosition => {
+                    setPosition(newPosition);
+                }}
+                onRotate={newRotation => {
+                    setRotation(newRotation);
+                }}
+            />
+            <MeshComponent
+                rootMesh={mesh}
+                allMeshes={mesh ? [mesh] : undefined}
+                mxPositionType={props.mxPositionType}
+                mxPositionXStat={props.mxPositionXStat}
+                mxPositionYStat={props.mxPositionYStat}
+                mxPositionZStat={props.mxPositionZStat}
+                mxPositionYAtt={props.mxPositionYAtt}
+                mxPositionXAtt={props.mxPositionXAtt}
+                mxPositionZAtt={props.mxPositionZAtt}
+                mxPositionXExpr={props.mxPositionXExpr}
+                mxPositionYExpr={props.mxPositionYExpr}
+                mxPositionZExpr={props.mxPositionZExpr}
+                mxRotationType={props.mxRotationType}
+                mxRotationXStat={props.mxRotationXStat}
+                mxRotationYStat={props.mxRotationYStat}
+                mxRotationZStat={props.mxRotationZStat}
+                mxRotationXAtt={props.mxRotationXAtt}
+                mxRotationYAtt={props.mxRotationYAtt}
+                mxRotationZAtt={props.mxRotationZAtt}
+                mxRotationXExpr={props.mxRotationXExpr}
+                mxRotationYExpr={props.mxRotationYExpr}
+                mxRotationZExpr={props.mxRotationZExpr}
+                mxScaleType={props.mxScaleType}
+                mxScaleXStat={props.mxScaleXStat}
+                mxScaleYStat={props.mxScaleYStat}
+                mxScaleZStat={props.mxScaleZStat}
+                mxScaleXAtt={props.mxScaleXAtt}
+                mxScaleYAtt={props.mxScaleYAtt}
+                mxScaleZAtt={props.mxScaleZAtt}
+                mxScaleXExpr={props.mxScaleXExpr}
+                mxScaleYExpr={props.mxScaleYExpr}
+                mxScaleZExpr={props.mxScaleZExpr}
+                OnSceneLoaded={handleSceneLoaded}
+                OnRotationSet={() => setRotationSet(true)}
+                mxMaterialColor={props.mxMaterialColor.value ?? "#0CABF9"}
```

Comment:
`"#0CABF9"` Magic value?

Reply:
It's just a starting color. To have something there. It should be Mendix blue :D